### PR TITLE
Check if go2rtc is even installed

### PR DESCRIPTION
**Pull request by dadav**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [ ] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)

This check will ensure that the go2rtc service will only be used if it was installed before. If not, the capture script will act like a wrapper for gphoto2.

### CODE REVIEW

File: install-photobooth.sh [hunk 1]
```diff
@@ -1075,9 +1075,13 @@ elif [[ \$# -gt 1 ]]; then
     args="\$@"
 fi
 
-sudo systemctl stop go2rtc.service
+if systemctl cat go2rtc.service >/dev/null; then
```
Comment (andi34) [inline install-photobooth.sh:1078]: Allowed for www-data user? Start and stop was explicitly rights given.

Comment (dadav) [reply install-photobooth.sh:1078]: Well, depends on the file permissions and the effective umask when the file was created. I don't think anyone has umask 077, so it should work...

Comment (dadav) [reply install-photobooth.sh:1078]: I think if a user doesn't modify the system it's fine

Comment (andi34) [reply install-photobooth.sh:1078]: Guess we can and should test this once, capture would fail with error. Can maybe give it a try later

Comment (dadav) [reply install-photobooth.sh:1078]: I tested it on my pi and it worked. Wasn't Bookworm though..

**Pull request merged**